assets/json/assets.json

```diff
@@ -41,6 +41,20 @@
                 "type" : "Anchored"
             },
             "children":  {
+                "innerRing":{
+                  "type": "Animation",
+                  "data": {
+                    "texture": "innerRing",
+                    "span": 91,
+                    "cols": 10,
+                    "frame": 0,
+                    "anchor": [0.5,0.5]
+                  },
+                  "layout": {
+                    "x_anchor" : "center",
+                    "y_anchor" : "middle"
+                  }
+                },
```

Comment:
Is there a reason why this needed to be added to the json file if we're just going to programmatically make new layer nodes in the code?